app/helpers/d3plus.js

```diff
@@ -77,7 +77,7 @@ export default {
     }
   },
   shapeConfig: {
-    hoverOpacity: 0.1,
+    hoverOpacity: 0.7,
```

Comment:
Selectively adjusting this value would be great or rather selectively overriding the default. I personally found the 0.7 value to be too jarring on hover to the point where I also had trouble reading the other data that in most cases I was trying to compare with the element I was hovering on. Can we split the difference with a 0.4?